multi_token_support_program/src/main.leo

```diff
@@ -59,17 +60,21 @@ program multi_token_support_program_v1.aleo {
   }
 
   async function finalize_initialize() {
+    // Check if the CREDITS_RESERVED_TOKEN_ID token has already been initialized
+    let already_initialized: bool = registered_tokens.contains(CREDITS_RESERVED_TOKEN_ID);
+    assert_eq(already_initialized, false);
+
     // Initialize the CREDITS_RESERVED_TOKEN_ID token
     let credits_reserved_token: TokenMetadata = TokenMetadata {
       token_id: CREDITS_RESERVED_TOKEN_ID,
       name: 1095517519u128,
       symbol: 1095517519u128,
       decimals: 6u8,
-      supply: 1_500_000_000_000_000u128,
-      max_supply: 1_500_000_000_000_000u128,
-      admin: multi_token_support_program_v1.aleo,
+      supply: 0u128, // placeholder value; not used
+      max_supply: 10_000_000_000_000_000u128,
```

Comment:
`Issue 10` Comment is invalid as we now manage supply via `mtsp_credits.aleo` but we see 10B credits as a reasonable upper bound for the total Aleo that may be held in the MTSP